subsys/bootloader/Kconfig

```diff
@@ -17,9 +17,27 @@ config CUSTOM_DTS_OVERLAY_CMAKE_FILE
 	# variable is not yet available when this config is used.
 	default "${ZEPHYR_BASE}/../nrf/subsys/bootloader/dts/overlay.cmake"
 
+config SB_PRIVATE_KEY_HANDLING
+	bool
+
+choice SB_SIGNING
+	prompt "Firmware signing method"
+	default SB_SIGNING_PYTHON
+	config SB_SIGNING_PYTHON
+		bool "Sign with Python ecdsa library."
+		select SB_PRIVATE_KEY_HANDLING
+	
+	config SB_SIGNING_OPENSSL
+		bool "Sign with openssl command line tool."
```

Comment:
The reason I didn't make it the default is that I imagine it will not work as well on Windows because openssl will typically be available only in Git Bash, and the commands use pipes ('|').